data/boarding missions.txt

```diff
@@ -23,6 +23,118 @@ mission "Assisting Merchant"
 	# A mission with no destination will not be offered, so give it one.
 	# What the destination is doesn't matter, because you always 'decline' this.
 	destination Earth
+	
+mission "Assisting Pirates (Thief)"
+	assisting
+	repeat
+	to offer
+		random < 12
+	source
+		government Pirate
+	on offer
+		payment -100000
+		conversation
+			`After assisting the <origin> you find that 100000 credits have been removed from your account.`
+				decline
+	destination Greenrock
+
+mission "Assisting Pirates (Setback)"
```

Comment:
With these two missions, there's really no point in assisting pirates; either you lose 100k credits or gain 100, and you're more likely to lose than to gain.